theme/napari/page.html

```diff
@@ -18,6 +18,27 @@
 
     <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}">
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}">
+
+    {%- block styles -%}
+
+      {# Custom stylesheets #}
+      {%- block regular_styles -%}
+        {%- for css in css_files -%}
+          {% if css|attr("filename") -%}
+            {{ css_tag(css) }}
+          {%- else -%}
+            <link rel="stylesheet" href="{{ pathto(css, 1)|e }}" type="text/css" />
+          {%- endif %}
+        {% endfor -%}
+      {%- endblock regular_styles -%}
+
+      {#- Theme-related stylesheets -#}
+      {%- block theme_styles %}
+      {%- endblock -%}
+
+      {%- block extra_styles %}
+      {%- endblock -%}
+    {%- endblock styles -%}
```

Comment:
This is copied from the [Furo theme](https://github.com/pradyunsg/furo/blob/5cdb35c041dabc6dc651cbd4b2f84226f2d24528/src/furo/theme/furo/base.html#L51-L72) 😆

Reply:
if possible, could you put this directly in as a comment?